docs/guide/create-dapp.md

```diff
@@ -0,0 +1,261 @@
+# Create A Simple Dapp
+::: tip 
+We will be building this [app](https://metamask.github.io/test-dapp/)
+:::
+
+## Project Setup
+Before you setup make sure you've visited and gone through our [Getting Started Guide](http://localhost:8080/guide/getting-started.html#getting-started)
```

Comment:
```suggestion
Before you setup make sure you've visited and gone through our [Getting Started Guide](/guide/getting-started.html#getting-started)
```

Reply:
^is this for the dapp getting started guide, or the general getting started with metamask guide?

Reply:
This is the general getting started guide.

Reply:
I wanted to go back and make the general getting started more "hands on" than what is currently in the docs